AxonDeepSeg/segment.py

```diff
@@ -55,35 +55,24 @@ def segment_image(path_testing_image, path_model,
 		# Get type of model we are using
 		tmp_path, selected_model = os.path.split(path_model)
 
-		# If the input image is not png
-		if not acquisition_name.endswith(".png"):  
+		# Read image
+		img = imageio.imread(os.path.join(path_acquisition,acquisition_name))
 
-			# Read image for conversion
-			img = imageio.imread(os.path.join(path_acquisition,acquisition_name))
+		# Generate new tmp file with '.png' extension and save
+		img_name_original, file_extension = os.path.splitext(acquisition_name) 
+		img_name = 'tmp.png'
```

Comment:
this could break if the native image is already called tmp.png. Please use [the proper way of creating temp file](https://docs.python.org/2/library/tempfile.html).